teal/game/backgroundslice.tl

```diff
@@ -7,20 +8,23 @@ local record BackgroundSlice
     canvas: Canvas.SubCanvas
     screenX: integer
     screenY: integer
+    collision_slice: CollideableSlice
 end
 
 local BackgroundSliceMetatable: metatable<BackgroundSlice> = {
     __index = BackgroundSlice
 }
 
 function BackgroundSlice:draw()
+    self.collision_slice:draw(self.canvas, 0, 0)
```

Comment:
After you've merged in main and/or rebased, the second argument to this should probably be `screenY`